man/networkd.conf.xml

```diff
@@ -138,6 +138,13 @@
             </para></listitem>
           </varlistentry>
 
+          <varlistentry>
+            <term><option>custom</option></term>
+            <listitem><para>If <literal>DUIDType=custom</literal>, then the <literal>DUIDRawData=</literal> value will
+            used be as custom identifier. If <literal>DUIDType=custom</literal> is specified then the
+            <literal>DUIDRawData=</literal> key is mandatory. Note it applies only on DHCPv6 clients.</para></listitem>
```

Comment:
I cannot find the corresponding code that the type is restricted to DHCPv6...

Reply:
Request is for DHCPv6 Not for DHCPv4.See https://github.com/systemd/systemd/issues/26745

Reply:
I see. But there is no reason to limit specifying custom DUID only for DHCPv6.